ssl/ssl_rsa.c

```diff
@@ -914,10 +914,8 @@ int SSL_CTX_use_serverinfo_file(SSL_CTX *ctx, const char *file)
     long extension_length = 0;
     char *name = NULL;
     char *header = NULL;
-    char namePrefix1[] = "SERVERINFO FOR ";
-    char namePrefix2[] = "SERVERINFOV2 FOR ";
-    unsigned int namePrefix1_len;
-    unsigned int namePrefix2_len;
+    static char namePrefix1[] = "SERVERINFO FOR ";
+    static char namePrefix2[] = "SERVERINFOV2 FOR ";
```

Comment:
`static const char`. The _const_ allows the compiler to place the entire string into readonly memory and optimize away the variable entirely.